### ISSUE TITLE
Cannot instantiate abstract commands

### ISSUE THREAD

**Issue by ejunker** [cronboard-io/cronboard-laravel#4]
I have a base command  `CronCommand` and some of my commands extend the base command which is an abstract class. The record command is trying to instantiate the abstract base command and getting this error.

```
❯ php artisan -v cronboard:record
Could not record command: Target [App\Console\CronCommand] is not instantiable.

   Cronboard\Commands\CommandBuilderException  : Could not record command: Target [App\Console\CronCommand] is not instantiable.

  at /project/vendor/cronboard-io/cronboard-laravel/src/Commands/Builder.php:45
    41|     {
    42|         try {
    43|             return $factory->build($target);
    44|         } catch (BindingResolutionException $e) {
  > 45|             throw new CommandBuilderException('Could not record command: ' . $e->getMessage(), $e->getCode(), $e);
    46|         }
    47|     }
    48|
    49|     protected function appendSupportData(Command $command = null): ?Command
```

I think you can use ReflectionClass to know if a class is abstract
```
$class = new ReflectionClass($className);
$class->isAbstract()
```

For now I will try to add it to the `ignore` in the config

**Comment by skovachev** [comment]
@ejunker , that's a good point - I'll see to it that such cases are handled automatically. 

And yes - adding it to the ignore section of the config file should solve it for the time being. 👍 
```
return [
    // ....
    'discovery' => [
        // ....
        'ignore' => [
            App\Console\CronCommand::class,
        ],
       // ....
    ]
];
```

**Comment by skovachev** [comment]
Fixed in 0.6.2
